src/app/modules/main/profile_section/sync/view.nim

```diff
@@ -40,21 +40,21 @@ QtObject:
   QtProperty[bool] automaticSelection:
     read = isAutomaticSelection
 
-  proc activeMailserverChanged*(self: View) {.signal.}
+  proc activeMailserverChanged*(self: View, activeMailserver: string) {.signal.}
```

Comment:
@richard-ramos please revert this to the previous state, `activeMailserver` is already exposed to the qml as `QtProperty` and sending just a `self.activeMailserverChanged()` signal from `proc onActiveMailserverSet` is enough to notify qml that value contained in `activeMailserver` has changed and all places which are using it will be updated to the new value, so there is no need for sending new value explicitly via signal and connecting to it on the qml side.